dialog/base.go

```diff
@@ -39,6 +40,14 @@ type dialog struct {
 	parent         fyne.Window
 }
 
+// SetOnClosed allows to set a callback function that is called when
+// the dialog is closed
+func (d *dialog) SetOnClosed(closed func()) {
+	d.callback = func(dummy bool) {
```

Comment:
One sticking point... It could be possible that this field is set already (like in a confirm dialog).
I think that probably this callback should call the other, if it exists (i.e. save a reference to it and call before the closed() callback)?

Reply:
Having a parameter named `dummy` is a waste of space. You can simply omit the parameter name if you don't intend to use it. If you want to express more explicitly that you didn't forgot the name but intentionally don't use the param, you can use a `_`.